validators/sql_schema_validator.py

```
@@ -0,0 +1,172 @@
+    "full",
+    "inner",
+    "outer",
+    "join",
+    "cross",
+    "union",
+}
+
+
+def _normalize_identifier(identifier: str) -> str:
+    ident = identifier.strip().strip("`")
+    # BigQuery table paths can be project.dataset.table – use the last segment
```

Ruff (0.13.1)

Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)